soh/soh/OTRGlobals.cpp

```diff
@@ -573,10 +573,13 @@ extern "C" uint32_t ResourceMgr_IsGameMasterQuest() {
         default:
             SPDLOG_WARN("Unknown rom detected. Defaulting to Non-mq {:x}", version);
             return 0;
-
     }
 }
 
+extern "C" uint32_t ResourceMgr_IsGameMasterQuest() {
+    return IsGameMasterQuest();
+}
+
```

Comment:
Needed a check for Master Quest in a CPP file, this is exported only to C files, while `IsGameMasterQuest` is usable in here and available to both. Allows for keeping the existing function while keeping code DRY.